android/java/org/chromium/chrome/browser/crypto_wallet/permission/BraveDappPermissionPromptDialog.java

```diff
@@ -209,7 +210,7 @@ private void setFavIcon() {
         }
         mRequestId = mWebContents.downloadImage(new GURL(mFavIconURL), // url
                 true, // isFavicon
```

Comment:
I don't think it matters, but perhaps we should pass false here as it's not a fav icon that we want to download

Reply:
ah, my bad, it's an old code here and it's a favicon actually :)

Reply:
The only difference when `true` (according to Javadoc) should be that does not send or accept cookies.
https://source.chromium.org/chromium/chromium/src/+/main:content/public/android/java/src/org/chromium/content_public/browser/WebContents.java;drc=31acc7339cf658ce82c7ec490ba38d67170920ed;l=472